### ISSUE TITLE
Dialog system painkillers

### ISSUE THREAD

**Issue by kerizane** [Tuxemon/Tuxemon#422]
**We could replace this:**

```
{
    "theinfo": "I'm sorry about that scene, Allie is sometimes hard to deal with, unlike my daughter Jess.",
    "theinfo2": "Oh! Where are my manners. My name is Ae... You know what? I'll just write it.",
    "theinfo2.5": "Aeble",
    "theinfo3": "It's pronounced ahbluh, but everyone just calls me Able. You can call me that, I guess...",
}

<property name="act1" value="translated_dialog theinfo"/>
<property name="act2" value="translated_dialog theinfo2"/>
<property name="act3" value="translated_dialog theinfo2.5"/>
<property name="act4" value="translated_dialog theinfo3"/>
<property name="act5" value="translated_dialog theinfo4"/>
```

**With this:**
```
"theinfo": [
        "I'm sorry about that scene, Allie is sometimes hard to deal with, unlike my daughter Jess.",
        "Oh! Where are my manners. My name is Ae... You know what? I'll just write it.",
        "Aeble",
        "It's pronounced ahbluh, but everyone just calls me Able. You can call me that, I guess...",
]

<property name="act1" value="translated_dialog theinfo"/>
```
We wouldn't need to create a new method or wrap everything in an array, as it would be easy to handle both types.

Note that the original went on for 17 lines.

Any other pain points?

**Comment by bitcraft** [comment]
This would be great!

**Comment by kerizane** [comment]
Other Issues:
<property name="act1" value="translated_dialog_chain professor_dialog1"/>
<property name="act7" value="dialog_choice Hydrone:Rockitten:Fruitera,tuxchoice"/>

Not ending the dialog chain results in the choice being called first.

**Comment by bitcraft** [comment]
Implemented in #423 Thank you @kerizane !
